portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Toggles the sort between ascending and descending, then updates comments */
+function toggleSort() {
+  const sortDirectionIcon = document.getElementById("sort-icon");
+  if (sortDirectionIcon.classList[0] === "sort-down") {
+    sortDirectionIcon.classList.remove('sort-down');
+    sortDirectionIcon.classList.add('sort-up');
+  } else {
+    sortDirectionIcon.classList.remove('sort-up');
+    sortDirectionIcon.classList.add('sort-down');
+  }
+  getComments();
+}
+
 /* Fetches comment json data data from /comment and displays them */
 function getComments() {
   const container = document.getElementById("comment-list");
 
-  fetch('/comment').then(response => response.json()).then(data => {
-    console.log("get data is", data);
+  const sortSelector = document.getElementById('comment-sort-select');
+  const maxSelector = document.getElementById('comment-max-select');
+  const sortDirectionIcon = document.getElementById("sort-icon");
+  
+
+  const sort = sortSelector.options[sortSelector.selectedIndex].value;
+  const max = maxSelector.options[maxSelector.selectedIndex].value;
+  const ascending = sortDirectionIcon.classList[0] === "sort-up" ? "true" : "false";
+
+  const query = "?sort=".concat(sort).concat("&max=").concat(max).concat("&ascending=").concat(ascending);
```

Comment:
I would suggest using a template literal in this case

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

It would look something like

```
`?sort=${sort}&max=${max}&ascending=${ascending}`
```